x-pack/metricbeat/module/panw/interfaces/_meta/fields.yml

```diff
@@ -0,0 +1,406 @@
+- name: interfaces
+  type: group
+  release: beta
+  description: >
+    interfaces
+  fields:
+    - name: physical.name
+      type: keyword
+      description: >
+        Physical interface name
+    - name: physical.id
+      type: keyword
+      description: >
+        Physical interface ID
+    - name: physical.type
+      type: keyword
+      description: >
+        Physical interface type (numeric value in source XML, string value sent to elasticsearch):
+          0:  "Ethernet interface",
+          1:  "Aggregate Ethernet (AE) interface",
+          2:  "High Availability (HA) interface",
+          3:  "VLAN interface",
+          5:  "Loopback interface",
+          6:  "Tunnel interface",
+          10: "SD-WAN interface",
+    - name: physical.mac
+      type: keyword
+      description: >
+        Physical MAC address
+    - name: physical.speed
+      type: keyword
+      description: >
+        Physical interface speed
+    - name: physical.duplex
+      type: keyword
+      description: >
+        Duplex configuration, e.g., "full" or "half"
+    - name: physical.state
+      type: keyword
+      description: >
+        Physical interface state: up/down
+    - name: physical.mode
+      type: keyword
+      description: >
+        Physical interface mode, e.g., autoneg
+    - name: physical.full_state
+      type: keyword
+      description: >
+        Physical full state, speed/duplex/state, e.g., "1000/full/up"
+    - name: physical.ae_member
+      type: keyword
+      description: >
+        For aggregate interfaces, the array of member interfaces
+    - name: logical.name
+      type: keyword
+      description: >
+        Logical interface name
+    - name: logical.id
+      type: keyword
+      description: >
+        Logical interface ID
+    - name: logical.tag
+      type: integer
+      description: >
+        VLAN tag associated with this interface
+    - name: logical.vsys
+      type: integer
```

Comment:
sure it is an integer? seems like it should be keyword based on description

Reply:
Apparently it is always an integer, representing values between 1 and 4094, as per the IEEE 802.1Q standard

Reply:
got it.